code/__HELPERS/tts.dm

```diff
@@ -0,0 +1,4 @@
+/proc/tts_alphanumeric_filter(text)
+	// Only allow alphanumeric characters and whitespace
+	var/static/regex/bad_chars_regex = regex("\[^a-zA-Z0-9 ,?.!'&-\\(\\)]", "g")
```

Comment:
what are you whitelisting for here exactly? unpronouncable things? why are & and () in here then?
related, have you considered using \l or \d  (or even \w? that might be too wide tho) <https://www.byond.com/docs/ref/#/{notes}/regex>

Reply:
changed the name so that the use is more clear, it's just a regex filter on tts speech (aka string identifiers of a tts voice)

Reply:
are the char classes worth using?

Reply:
I don't think there's any significant difference, plus they run the risk of being more wide and allowing in values that we don't want to allow.